src/lumigo_opentelemetry/__init__.py

```diff
@@ -77,6 +77,16 @@ def auto_load(_: Any) -> None:
 
 
 def init() -> Dict[str, Any]:
+    """Initialize the Lumigo OpenTelemetry distribution."""
+    python_version = sys.version_info
+    # Check if the major version is 3 and the minor version is between 8 and 11
+    if python_version.major != 3 or not (8 <= python_version.minor <= 11):
+        logger.warning(
+            f"Unsupported Python version {python_version.major}.{python_version.minor}; "
+            "only Python 3.8 to 3.11 are supported."
+        )
+        return {}
```

Comment:
Put this inside a try catch block, so if in python 3.13 they change the interface of `sys.version_info` then the tracer won't crash.

If this block fails just log a warning saying that we failed to verify the python version.

Reply:
Don't only check for an `AttributeError` check for any error.
This is a fail safe for unexpected errors, so don't catch only one expected error...

Reply:
Approved the fix